src/other/utils.jl

```diff
@@ -178,3 +178,155 @@ function _nt_like_hash(v, h::UInt)
 
     return xor(objectid(Tuple(propertynames(v))), h)
 end
+
+_findall(B) = findall(B)
+
+@inline _blsr(x)= x & (x-1)
+const _msk64 = ~UInt64(0)
+
+# findall optimized for B::BitVector
+# the main idea from Base.findall(B::BitArray)
+function _findall(B::BitVector)::Union{UnitRange{Int}, Vector{Int}}
+    nnzB = count(B)
+    nnzB == 0 && return Int[]
```

Comment:
How about this instead?
```suggestion
    nnzB == 0 && return 1:0
```

Reply:
I felt `Int[]` is not a good way, but I couldn't find better, thanks :)